src/controller/manualAttendance.js

```diff
@@ -0,0 +1,17 @@
+import db from '../db';
+
+module.exports = {
+
+    manualUpdatedAttendance: (req, res, next) => {
+        db.manual_attendance.manualUpdateAttendance(req, res).then((data) => { // gets all data manually entered by employees which is not reviewed
```

Comment:
response is always send through controller or a common response handler for all responses